src/app/ApplicationTemplate.Mobile/ApplicationTemplate.Mobile.csproj

```diff
@@ -109,7 +132,7 @@
 					<PropertyGroup>
 						<CodesignEntitlements>./iOS/Entitlements.plist</CodesignEntitlements>
 						<MtouchDebug>True</MtouchDebug>
-						<MtouchExtraArgs>$(MtouchExtraArgs) --optimize=-remove-dynamic-registrar --setenv=MONO_LOG_LEVEL=debug --setenv=MONO_LOG_MASK=gc</MtouchExtraArgs>
+						<MtouchExtraArgs>$(MtouchExtraArgs) --setenv=MONO_LOG_LEVEL=debug --setenv=MONO_LOG_MASK=gc</MtouchExtraArgs>
```

Comment:
I wonder if you won't face issues by removing registrar to deploy on iOS simulator ?

Reply:
None, only the static one was required for it to work.